submissions/kolawoleOgunfowokan/HTML/css/style.css

```diff
@@ -0,0 +1,140 @@
+* {
+
```

Comment:
Please remove this space everywhere else. It is not needed.

Reply:
Done, the spaces have been removed.